configuration/scripts/options/set_nml.box2000

```diff
@@ -0,0 +1,56 @@
+days_per_year = 360
```

Comment:
why is this file named box2000?

Reply:
Hunke 2000?  Happy to name it something else.  I assume that's where box2001 came from, Hunke 2001?

Reply:
Yes, box2001 is the test case used in Hunke (2001).  I only had one paper in 2000, and it was about penguin moulting habitat!!!  So let's come up with some other name.  Is this the 1D test in appendix A?  Maybe call it box1Dwall and put a comment in the code that it's appendix A in Hunke (2001), and note any differences, e.g. there's only ice in half the domain.